src/extension-runners/chromium.js

```diff
@@ -151,12 +151,26 @@ export class ChromiumExtensionRunner {
     });
   }
 
-  wssBroadcast(data: Object) {
+  async wssBroadcast(data: Object) {
+    const promisesArray = [];
     for (const client of this.wss.clients) {
       if (client.readyState === WebSocket.OPEN) {
-        client.send(JSON.stringify(data));
+        promisesArray.push(new Promise((resolve) => {
+          client.on('message', async (message) => {
```

Comment:
With this implementation, the number of `message` listeners will increase whenever reload is called.

I suggest to store the list of clients in an array, create one function, and then use that function as a listener for every `client`. When any of the clients return a message, all listeners should be unregistered, and the promise should be resolved. When a client is disconnected, the listener should be unregistered and the client be removed from the list (and if the list is empty, the promise should be resolved).